kyuubi-server/src/main/scala/org/apache/kyuubi/server/api/v1/AdminResource.scala

```diff
@@ -53,4 +58,64 @@ private[v1] class AdminResource extends ApiRequestContext with Logging {
     KyuubiServer.reloadHadoopConf()
     Response.ok(s"Refresh the hadoop conf for ${fe.connectionUrl} successfully.").build()
   }
+
+  @ApiResponse(
+    responseCode = "200",
+    content = Array(new Content(
+      mediaType = MediaType.APPLICATION_JSON)),
+    description = "delete kyuubi engine")
+  @DELETE
+  @Path("engine")
+  def deleteEngine(
+      @QueryParam("type") engineType: String,
+      @QueryParam("subdomain") engineSubdomain: String,
+      @QueryParam("sharelevel") engineShareLevel: String,
```

Comment:
Re-order and short the param name, `engineShareLevel` -> `shareLevel`, `engineSubdomain` -> `subdomain`
```
@QueryParam("sharelevel") shareLevel: String,
@QueryParam("subdomain") subdomain: String,
```